LanguageApp/app/screens/UserTypeScreen.js

```diff
@@ -4,19 +4,24 @@ import React from "react";
 import AppTitle from "../components/AppTitle";
 import Screen from "../components/Screen";
 import AppButton from "../components/AppButton";
+import routes from "../navigation/routes";
+import colors from "../config/colors";
 
-function UserTypeScreen() {
+function UserTypeScreen({ navigation }) {
   return (
     <Screen>
       <View style={styles.container}>
-        <AppTitle style={styles.title}>Choose User Type</AppTitle>
+        {/* <AppTitle style={styles.title}>Choose User Type</AppTitle> */}
```

Comment:
Same as previous comments. Can be removed since no longer using AppTitle component

Reply:
yes!